core/services/ocr2/plugins/vault/service.go

```diff
@@ -81,12 +81,20 @@ func (s *Service) Execute(ctx context.Context, request capabilities.CapabilityRe
 	}
 	id := fmt.Sprintf("%s::%s::%s", md.WorkflowID, phaseOrExecution, md.ReferenceID)
 
-	resp, err := handleRequest[*vault.GetSecretsResponse](ctx, s, id, r)
+	resp, err := handleRequest(ctx, s, id, r)
 	if err != nil {
 		return capabilities.CapabilityResponse{}, err
 	}
 
-	anyproto, err := anypb.New(resp)
+	// Note: we can drop the signatures from the response above here
```

Comment:
Is this just informational comment, or do we intend to also drop the signatures?